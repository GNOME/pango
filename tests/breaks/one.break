abc/def ghi­jkl. Bla
