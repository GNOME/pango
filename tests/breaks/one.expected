Text:         a b c  /  d e f  [ ]  g h i [0xad]  j k l  . [ ]  B l a  [0x0a] 
Breaks:     c  c c c  lc c c c    lc c c c      lc c c c  c   lc c c c       c
Whitespace:                  x                            x          w       w
Words:      bs     be bs     be   bs                   be b   bs     be      b
Sentences:  bs                                            e   bs     e       b
